rundeckapp/grails-app/services/rundeck/services/FrameworkService.groovy

```diff
@@ -57,6 +52,7 @@ import java.util.function.Predicate
 class FrameworkService implements ApplicationContextAware, AuthContextProcessor, ClusterInfoService {
     static transactional = false
     public static final String REMOTE_CHARSET = 'remote.charset.default'
+    public static final List PLUGINS_TYPES = ['Notification','WorkflowNodeStep', 'RemoteScriptNodeStep']
```

Comment:
why is it only listing these plugin types?

Reply:
Actually, it is wrong. Should have all groovy and script-based plugin type